### ISSUE TITLE
Unable to update modules via update_groups.sh

### ISSUE THREAD

**Issue by uneti-max** [terraform-aws-modules/terraform-aws-security-group#266]
## Description

Hoi All 👋 

I'm trying to add a new set of rules + auto-group to rules.tf to enable using Loki and promtail on an AWS ec2 instance. 

I have added the following to rules.tf

```hcl
# Loki
loki          = [3100, 3100, "tcp", "Grafana Loki enpoint"]
loki-grpc     = [9096, 9096, "tcp", "Grafana Loki GRPC"]
loki-promtail = [9200, 9200, "tcp", "Promtail endpoint"]
```

```hcl
loki = {
  ingress_rules     = ["loki", "loki-grpc", "loki-promtail"]
  ingress_with_self = ["all-all"]
  egress_rules      = ["all-all"]
    }
```

After doing this, i change directories to `terraform-aws-security-group` and then run `update_groups.sh`

The output I receive is:

```bash
./update_groups.sh
jq: error (at <stdin>:1655): Cannot index array with string "default"
jq: error (at <stdin>:1655): Cannot index array with string "default"
There are no modules to update. Check values of auto_groups inside rules.tf
```

- I'm executing the script from an M1 mac running Ventura 13.1.
- sed did not need to be installed, was already present on system
- hcl2json and jq have been installed via Brew

```bash
brew list hcl2json 
/opt/homebrew/Cellar/hcl2json/0.3.5/bin/hcl2json

brew list jq
/opt/homebrew/Cellar/jq/1.6/bin/jq
/opt/homebrew/Cellar/jq/1.6/include/ (2 files)
/opt/homebrew/Cellar/jq/1.6/lib/libjq.1.dylib
/opt/homebrew/Cellar/jq/1.6/lib/ (2 other files)
/opt/homebrew/Cellar/jq/1.6/share/doc/ (4 files)
/opt/homebrew/Cellar/jq/1.6/share/man/man1/jq.1
```

## Versions

- Module version [Required]: 4.16.1

- Terraform version: Terraform v1.3.4

## Reproduction Code [Required]

```bash
./update_groups.sh
```

## Steps to reproduce the behavior:

- Clone this repo
- Edit rules.tf with Loki stanzas above
- Run ./update_groups.sh

## Expected behavior

A new module named Loki should be created and populated

## Actual behavior

```bash
./update_groups.sh
jq: error (at <stdin>:1655): Cannot index array with string "default"
jq: error (at <stdin>:1655): Cannot index array with string "default"
There are no modules to update. Check values of auto_groups inside rules.tf
```

Hopefully its just a 'my-machine' sort of issue, any help is appreciated <3

Thanks in advance, and cheers!

**Comment by antonbabenko** [comment]
Fixed in 1e33b5f. I was using the incorrect version of `hcl2json` locally. Please try now.

**Comment by uneti-max** [comment]
awesome, thanks!
